docs/content/concepts/annotation-context.md

```diff
@@ -49,11 +49,11 @@ Keypoints are currently only applicable to 2D and 3D points.
 Annotation Context is typically logged as [timeless](timelines.md#timeless-data) data, but can change over time if needed.
 
 The Annotation Context is defined as a list of Class Descriptions that define how classes are styled
-(as well as optional keypoint style & connection).
+(as well as optional keypoint style and connection).
 
 Annotation contexts are logged with:
 
-* Python: [`log_annotation_context`](https://ref.rerun.io/docs/python/latest/common/annotations/#rerun.log_annotation_context)
+* Python: [`rr.AnnotationContext`](https://ref.rerun.io/docs/python/HEAD/common/annotations/#rerun.AnnotationContext)
 * Rust: [`AnnotationContext`](https://docs.rs/rerun/latest/rerun/components/struct.AnnotationContext.html)
```

Comment:
Is this correct?

Reply:
if we write `rr.AnnotationContext` we should probably also write `rerun::AnnotationContext` for Rust 🤔

Reply:
oh and the link will drop the `components` part soon. You can link to the alpha to get a working link

Reply:
ah shoot we don't have it on the rerun namespace on the alpha yet https://docs.rs/rerun/0.9.0-alpha.6/rerun/archetypes/struct.AnnotationContext.html#